Source/multi.cpp

```diff
@@ -76,6 +74,14 @@ const event_type EventTypes[3] = {
 
 namespace {
 
+struct TBuffer {

```

Comment:
Likewise, we don't serialize this directly, right?

Reply:
Dito